groupnews-web/src/main/resources/logback-test.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this config for test context? It should be simply logback.xml to also ensure logging when you start the Boot app.